Apps/Laboratory/test/unit/Services/LaboratoryService_Test.cs

```diff
@@ -0,0 +1,170 @@
+//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+// <auto-generated />
```

Comment:
New code shouldn't have the auto-generated as it should comply with our standards.

Reply:
oh I have copied and pasted from other tests :-) Should we completely remove this session then?

Reply:
updated tests